canopy_content.ml

```diff
@@ -49,14 +49,24 @@ let to_atom = function
 
 let find_tag tagname = function
   | Markdown m ->
-     List.exists ((=) tagname) m.Canopy_article.tags
+    List.exists ((=) tagname) m.Canopy_article.tags
 
 let date = function
   | Markdown m ->
-     m.Canopy_article.created
+    m.Canopy_article.created
 
 let compare a b = Ptime.compare (date b) (date a)
 
 let updated = function
   | Markdown m ->
     m.Canopy_article.updated
+
+let tags content_map =
```

Comment:
a new function that returns all existings tags (as string list) from the cache map (populated in Store.fill_cache)